frontend_app_admin_workspace_user/build_app.sh

```diff
@@ -23,16 +23,27 @@ function logerror {
 
 
 dev=""
-if [ "$1" = "-d" ]; then
-    dev=":dev"
-fi
+only_utils=""
+
+while [[ "$#" -gt 0 ]]; do
+    case $1 in
```

Comment:
Probably best to quote this:

```
    case "$1" in
```

Reply:
will fix

Reply:
fixed
also fixed un run_dev_backend.sh

Reply:
Yes, haven't clarified but I only commented on the first occurrence for syntax concerns.